.github/workflows/php-stan.yml

```diff
@@ -0,0 +1,37 @@
+name: phpstan
+on:
+  push:
+    branches:
+      - master
+      - release_3_4
+      - release_3_3
+  pull_request:
+    branches:
+      - master
+      - release_3_4
+      - release_3_3
+
+defaults:
+  run:
+    working-directory: ./lizmap
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup PHP with tools
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          tools: phpstan
+
+      - name: Install dependencies
+        run: composer install --prefer-dist --no-progress --no-suggest
```

Comment:
is there a way in github workflows to configure a cache of files downloaded  by Composer (or any other packager like npm etc ?)

Reply:
It's possible to setup cache on GH Action.
We can look to that after ?